src/vs/workbench/parts/emmet/node/actions/expandAbbreviation.ts

```diff
@@ -31,7 +31,8 @@ class ExpandAbbreviationAction extends BasicEmmetEditorAction {
 					EditorContextKeys.hasOnlyEmptySelection,
 					EditorContextKeys.hasSingleSelection,
 					EditorContextKeys.tabDoesNotMoveFocus,
-					ContextKeyExpr.has('config.emmet.triggerExpansionOnTab')
+					ContextKeyExpr.has('config.emmet.triggerExpansionOnTab'),
+					ContextKeyExpr.not('config.emmet.autocomplete')
```

Comment:
Shouldn't we have a schema definition for this? Like for other values: https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/parts/emmet/node/emmet.contribution.ts#L38